Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7.9-slim-buster as base
+
+ENV PATH /usr/local/bin:$PATH
+ENV VIRTUAL_ENV "/venv"
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH "$VIRTUAL_ENV/bin:$PATH"
+WORKDIR /app
+RUN pip install poetry
```

Comment:
Minor, but it's a bit nicer to use the official route for installing poetry rather than depending on pip (i.e. [see the docs](https://python-poetry.org/docs/))